### ISSUE TITLE
Partial transparency of default-bg ignored

### ISSUE THREAD

**Issue by sebastinas** [pwmt/zathura#157]
*On GitLab by @Synthetica on Oct 11, 2019, 13:47*

---

# Issue description

(Running Zathura under Wayland)

To reproduce: `:set default-bg "rgba(255,255,255,10)"`

Expected result: a background that is slightly white, with my wallpaper shining through.

Actual result: a fully white wallpaper.

Alternate case `:set default-bg "rgba(255,255,255,0)"` does work as expected, and my wallpaper comes through completely: https://i.imgur.com/MantekM.png 

Alacritty on Sway does support regional partial transparency, so it doesn't seem to be a Sway issue.

# Versions

```
$ sway --version
sway version 1.2
$ uname -a
Linux AquaRing 5.3.5 #1-NixOS SMP Mon Oct 7 17:01:58 UTC 2019 x86_64 GNU/Linux
$ zathura --version
warning: Unknown option: smooth-scroll
zathura 0.4.4
girara 0.3.3 (runtime: 0.3.3)
(plugin) djvu (0.2.8) (/nix/store/xr6f1rcpm1a3n39dx5h6982sp1k61zq3-zathura-djvu-0.2.8/lib/zathura/libdjvu.so)
(plugin) ps (0.2.6) (/nix/store/ffkklx737cw5laay2rgica37alv21fwq-zathura-ps-0.2.6/lib/zathura/libps.so)
(plugin) cb (0.1.8) (/nix/store/y4ziqbn7s0b5x629sbwb71niks5ajclb-zathura-cb-0.1.8/lib/zathura/libcb.so)
(plugin) pdf-mupdf (0.3.5) (/nix/store/j24mpglvk5rpskiwll4znpric0x0d53w-zathura-pdf-mupdf-0.3.5/lib/zathura/libpdf-mupdf.so)
```

**Comment by sebastinas** [comment]
This seems to be a little weird on the Gdk side. `rgba(r,g,b,a)` is interpreted with `r`, `g` `b` being integers between 0 and 255, and `a` being a float between `0` and `1`. In your case, the `alpha` of 1 is capped at `1` which means full opacity.

**Comment by sebastinas** [comment]
closed via commit a3e4bce3860ae9962b4d3254cbb9473d34096c8e

**Comment by sebastinas** [comment]
mentioned in commit a3e4bce3860ae9962b4d3254cbb9473d34096c8e
